api/api.py

```diff
@@ -249,14 +246,14 @@ async def compare_in_repo(ids: str):
                 # diff repo, diff usage scenarios, same machine,  same branches, diff/same commits_hashes
                 phase_stats_object['common_info']['Machine'] = machine
                 phase_stats_object['common_info']['Branch'] = branch
-                # TODO: diff/same commit?
             case 'Branch':
                 # same repo, same usage scenarios, same machines, diff branch
                 phase_stats_object['common_info']['Repository'] = uri
                 phase_stats_object['common_info']['Usage Scenario'] = usage_scenario
                 phase_stats_object['common_info']['Machine'] = machine
 
     except RuntimeError as err:
+        # FIXME: 204? 500?
```

Comment:
@ArneTR Some of the comparison helper functions raise `RuntimeError` in cases where there is no data or invalid comparison case. I can't distinguish if it should be a HTTP 204 or maybe a 400 or even 500?